src/feature/flow_chart_panel/views/NodeModal.tsx

```diff
@@ -18,30 +189,111 @@ const NodeModal = ({
   nodeLabel,
   nodeType,
   nd,
+  nodeFileName,
   pythonString,
   clickedElement,
 }: NodeModalProps) => {
   const theme = useMantineTheme();
+  const { classes } = NodeModalStyles();
+  const { darkFlojoy, lightFlojoy } = useFlojoySyntaxTheme();
+  const { lightJSONTree, darkJSONTree } = themeJSONTree(theme);
 
+  const GLINK = "https://github.com/flojoy-io/nodes/blob/main";
+  let nodeCategory = "";
+  const nodeDataLabel = nodeFileName.split(".")[0];
   const colorScheme = theme.colorScheme;
+
+  let LINK = `${GLINK}`;
+  if (getPath(CMND_TREE, nodeType) !== null) {
+    const path: string[] = getPath(CMND_TREE, nodeType)!;
+    let pathLength = path.length;
+    if (nodeType === "STEPPER") pathLength = -1;
+    nodeCategory = path[0];
+    for (let i = 0; i < pathLength; i++) {
+      LINK += `/${path[i]}`;
+    }
+  }
+
+  switch (nodeCategory) {
```

Comment:
Can we set a name convention for all categories/sub-categories/files so that we can get rid of this `switch` method and we don't have to change here again in case of new category/file is introduced?